test/basic.html

```diff
@@ -90,7 +91,11 @@
         input.addEventListener('blur', function(event) {
           assert(!input.focused, 'input is blurred');
           assert.notEqual(getComputedStyle(error).visibility, 'hidden', 'error is not visibility:hidden');
-          done();
```

Comment:
I would prefer it matched the similar test in `paper-input`: https://github.com/PolymerElements/paper-input/blob/master/test/paper-input.html#L199